java/code/src/com/redhat/rhn/common/db/datasource/xml/ChannelReport_queries.xml

```diff
@@ -20,23 +20,76 @@
                         END AS type
-                      , rhnchannelarch.name AS arch
+                      , rhnchannelarch.label AS arch
+                      , rhnchecksumtype.label AS checksum_type
                       , rhnchannel.summary
                       , rhnchannel.description
                       , rhnparentchannel.label AS parent_channel_label
-                      , web_customer.name AS organization 
+                      , rhnchannelcloned.original_id AS original_channel_id
+                      , web_customer.name AS organization
             FROM rhnchannel
                       LEFT JOIN rhnchannelarch ON rhnchannel.channel_arch_id = rhnchannelarch.id
+                      LEFT JOIN rhnchecksumtype on rhnchecksumtype.id = rhnchannel.checksum_type_id
                       LEFT JOIN rhnchannel rhnparentchannel ON rhnchannel.parent_channel = rhnparentchannel.id
+                      LEFT JOIN rhnchannelcloned ON rhnchannel.id = rhnchannelcloned.id
                       LEFT JOIN web_customer ON rhnchannel.org_id = web_customer.id
         ORDER BY channel_id OFFSET :offset LIMIT :limit
   </query>
 </mode>
 
+<mode name="ChannelPackage" class="">
+    <query params="offset, limit">
+          SELECT rhnchannel.id AS channel_id
+                      , rhnchannel.label AS channel_label
+                      , rhnpackage.id AS package_id
+                      , rhnpackagename.name AS package_name
+                      , rhnpackageevr.epoch AS package_epoch
+                      , rhnpackageevr.version AS package_version
+                      , rhnpackageevr.release AS package_release
+                      , rhnpackageevr.type AS package_type
+                      , rhnpackagearch.label AS package_arch
+            FROM rhnchannel
+                      INNER JOIN rhnchannelpackage ON rhnchannel.id = rhnchannelpackage.channel_id
+                      INNER JOIN rhnpackage ON rhnpackage.id = rhnchannelpackage.package_id
+                      INNER JOIN rhnpackagename ON rhnpackage.name_id = rhnpackagename.id
+                      INNER JOIN rhnpackageevr ON rhnpackage.evr_id = rhnpackageevr.id
+                      INNER JOIN rhnpackagearch ON rhnpackage.package_arch_id = rhnpackagearch.id
+        ORDER BY channel_id, package_id OFFSET :offset LIMIT :limit
+    </query>
+</mode>
+
+<mode name="ChannelErrata" class="">
+    <query params="offset, limit">
+          SELECT rhnchannel.id AS channel_id
+                    , rhnchannel.label AS channel_label
+                    , rhnerrata.id AS errata_id
+                    , rhnerrata.advisory_name
+            FROM rhnchannelerrata
+                    INNER JOIN rhnchannel ON rhnchannelerrata.channel_id = rhnchannel.id
+                    INNER JOIN rhnerrata ON rhnchannelerrata.errata_id = rhnerrata.id
+        ORDER BY channel_id, errata_id OFFSET :offset LIMIT :limit
+    </query>
+</mode>
+
+<mode name="ChannelContentSource" class="">
```

Comment:
I would like to not name repositories "ContentSource" in the reportdb.

Can we rename it to "Repository"? It is a more common name so everybody knows what he can expect.
Of cause we cannot rename this in the main DB, but when we introduce this in the reportdb we should use better names if we do not like the original once.

Reply:
Every reference to ContentSource has been renamed to Repository.